layers/+lang/common-lisp/packages.el

```diff
@@ -33,6 +34,10 @@
 (defun common-lisp/post-init-helm-gtags ()
   (spacemacs/helm-gtags-define-keys-for-mode 'common-lisp-mode))
 
+(defun common-lisp/init-slime-company ()
+  :config
```

Comment:
I use `:config` to delay setting of the variable until the package is loaded. That is, when the variable is available.